modules/plugin/geopkg/src/main/java/org/geotools/geopkg/GeoPkgDialect.java

```diff
@@ -97,14 +102,19 @@ public class GeoPkgDialect extends PreparedStatementSQLDialect {
 
     private JSONArrayIO jsonArrayIO = new JSONArrayIO();
 
+    // geopkg - format will be ISO-8601 - YYYY-MM-DDTHH:MM[:SS.SSS]Z
+    SimpleDateFormat geopkgDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'");
```

Comment:
Nice, I never remember if format is thread safe or can be a constant.

Reply:
Not thread safe